indra/newview/skins/default/xui/en/notifications.xml

```diff
@@ -8837,10 +8837,11 @@ New Voice Morphs are available!
     persist="true"
     type="alertmodal">
 Voice Morphs are not supported by this viewer.
+For more information about other voice morph tools, see [[FAQ_URL] this article].
     <usetemplate
-        notext="Cancel"
-        name="okcancelbuttons"
-        yestext="Disable Voice Morphing"/>
+        ignoretext="Don't show me this again"
```

Comment:
`igoretext` isn't a notification text. This text appears in preferences floater instead and should describe what it does, like "Warn me about voice morph not being supported"

Reply:
ohh gotcha, thanks

Reply:
I went ahead and merged this after removing this line.  please let me know if there's anything else to follow up with furthermore

Reply:
Make sure (or note for QA to make sure) that there is no weird "$ignoretext" in preferences->notifications now, since that the default value declared in "okignore", if you want that checkbox to not be in settings, you need to set it `checkbox_only="true"`